### DISCUSSION TITLE
Page 'Design RESTful Services' Return Type was wrong in ServiceStack document

### DISCUSSION THREAD

**Discussion by mylvghb** [ServiceStack/Discuss#86] [General]
URL: https://docs.servicestack.net/design-rest-services#implementing-restful-routes

Returns naked value is `List<EventReview>`:

```csharp
[Route("/events/{EventId}/reviews", "GET")]
public class GetEventReviews : IReturn<List<EventReview>>
{
    public int EventId { get; set; }
}
```
But, in `GetEventReviewsResponse` is `List<Event>`:

```csharp
[Route("/events/{EventId}/reviews", "GET")]
public class GetEventReviews : IReturn<GetEventReviewsResponse>
{
    public int EventId { get; set; }
}

public class GetEventReviewsResponse
{
    public List<Event> Results { get; set; }
}
```

**Comment by mythz** [comment]
Thx for reporting, updated in [this commit](https://github.com/ServiceStack/docs.servicestack.net/commit/c8bf2be3310823de9dbaedbabe40d24b1f46e511).

FYI all docs have a **Edit this page on GitHub** link at the bottom of the page where you can quickly edit the page to submit a PR with fixes like this, which is a great way to report issues with docs.

**Reply by mylvghb** [reply]
I've submitted a [PR](https://github.com/ServiceStack/docs.servicestack.net/pull/2) at docs.servicestack.net
